dockerfiles/ocw/node-hugo/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM  node:14-buster-slim
+FROM  node:20-buster-slim
 LABEL maintainer="MIT Open Learning (odl-devops@mit.edu)"
 LABEL description="Node and Hugo, used for building OCW website"
 LABEL version="0.6"
```

Comment:
Can we up the version number here so current pipelines don't immediately start using node 20?

Reply:
:+1: